src/components/v2/V2Project/V2ProjectReconfigureModal/drawers/V2ReconfigureFundingDrawer.tsx

```diff
@@ -1,32 +1,28 @@
 import { Trans } from '@lingui/macro'
 import { Drawer } from 'antd'
-import StandardSaveButton from 'components/StandardSaveButton'
-import FundingTabContent from 'components/v2/V2Create/tabs/FundingTab/FundingTabContent'
 
 import { drawerStyle } from 'constants/styles/drawerStyle'
 import { FundingDrawersSubtitles } from '..'
 
 export function V2ReconfigureFundingDrawer({
   visible,
-  onSave,
   onClose,
+  title,
+  content,
 }: {
   visible: boolean
-  onSave: VoidFunction
   onClose: VoidFunction
+  title: JSX.Element
+  content: JSX.Element
 }) {
   return (
     <Drawer visible={visible} {...drawerStyle} onClose={onClose}>
       <h3>
-        <Trans>Reconfigure funding target/duration</Trans>
+        <Trans>{title}</Trans>
```

Comment:
**issue**: shouldn't wrap in `Trans` tag if there's no static text

Reply:
Ahh yep good catch